clusterdock/utils.py

```diff
@@ -200,3 +207,28 @@ def print_topology_meta(topology_name, quiet=False):
             logger.info('%s has Git hash %s', topology_name, out)
     except:
         pass
+
+
+def get_containers(clusterdock=False):
+    """
+    Get Docker containers.
+
+    Args:
+        clusterdock (:obj:`bool`, optional): Clusterdock containers only. Default: ``False``
```

Comment:
`Clusterdock` => `clusterdock`

Reply:
starting of sentence, so thats why caps. ok with either one - so fixed.